setup.py

```diff
@@ -106,8 +106,9 @@
     'License :: OSI Approved :: Apache Software License',
 ]
 
-BUILD_WITH_BORING_SSL_ASM = os.environ.get('GRPC_BUILD_WITH_BORING_SSL_ASM',
-                                           True)
+# Environment variable to turn off BoringSSL ASM optimization.
+BUILD_WITHOUT_BORING_SSL_ASM = os.environ.get(
```

Comment:
Isn't this a breaking change? What if we just add our own layer of "truthiness conversion" to this? Something like

```python
def _is_truthy(s):
  return s.lower() == "true" or s.lower() == "t" or s == "1"

BUILD_WITH_BORING_SSL_ASM = _is_truthy(os.environ.get('GRPC_BUILD_WITH_BORING_SSL_ASM',
                                           "True"))
```
This keeps basically the same interface (which people may already be depending on) and allows people to selectively disable if desired.

Reply:
Yeah. We should have kept the same key name pattern. But we should not break it as well.